src/ralph/admin/mixins.py

```diff
@@ -203,6 +203,19 @@ def formfield_for_manytomany(self, db_field, request=None, **kwargs):
             return db_field.formfield(**kwargs)
         return super().formfield_for_manytomany(db_field, request, **kwargs)
 
+    def get_changelist_form(self, request, **kwargs):
```

Comment:
could you add some docstring here (what this method do, why this weird class `vTextField no-fillable` etc)?